docs/guides/deployment/kubernetes.md

```diff
@@ -10,9 +10,11 @@ search:
   boost: 2
 ---
 
-# Running flows with Kubernetes
+# Running flows with Kubernetes and a Kubernetes worker
 
-This guide will walk you through running your flows on Kubernetes. Though much of the guide is general to any Kubernetes cluster, there are differences between the managed Kubernetes offerings between cloud providers, especially when it comes to container registries and access management. We'll focus on Amazon Elastic Kubernetes Service (EKS).
+This guide will walk you through running your flows on Kubernetes. 
```

Comment:
Looks like each sentence got assigned to a different line. Is this intended/preferred?

Reply:
It was intentional. I have read/heard that this is best practice for easier reviews and edits. It's not something we've standardized on. We should decide whether we want to do it that way.

Reply:
I think it makes sense... should track changes more atomically.